docs/getting-started.md

```diff
@@ -0,0 +1,119 @@
+# Getting Started
+
+Before diving into Langium itself, let's get your environment ready for development:
+
+1. You have a working [Node environment](https://nodejs.org/en/download/) with version 12 or higher.
+2. Install Yeoman and the Langium extension generator.
+```bash
+npm i -g yo generator-langium
+```
+
+For our getting started example, we would also recommend you to install the latest version of [vscode](https://code.visualstudio.com/).
+
+## Your first example language
+
+To create your first working DSL, execute the yeoman generator:
+
+```bash
+yo langium
+```
+
+Yeoman will prompt you with a few basic questions about your DSL:
+
+1. _Extension name_: Will be used as the folder name of your extension and its `package.json`.
+2. _Language name_: Will be used as the name of the grammar and as a prefix for any generated files. 
+3. _File extensions_: A comma separated list of file extensions for your DSL.
+
+Afterwards, it will generate a new project and start installing all dependencies, including the `langium` framework as well as the `langium-cli` command line tool required for generating code based on your grammar definition.
+
+After everything has successfully finished running, open your newly created Langium project with vscode (replacing `hello-world` with your chosen project name):
+
+```bash
+code hello-world
+```
+
+Press F5 or open the debug view and start the available debug configuration to launch the extension in a new _Extension Development Host_ window. Open a folder and create a file with your chosen file extension (`.hello` is the default). The `hello-world` language accepts two kinds of entities: The `person` and `Hello` entity. Here's a quick example on how to use them both:
+
+```
+person Alice
+Hello Alice!
+
+person Bob
+Hello Bob!
+```
+
+And here's the Langium grammar that parses this file:
+
+```
+grammar HelloWorld hidden(WS)
+
+terminal WS: /\s+/;
+terminal ID: /[_a-zA-Z][\w_]*/;
+
+Model: (persons+=Person | greetings+=Greeting)*;
+
+Person:
+    'person' name=ID;
+
+Greeting:
+    'Hello' person=[Person] '!';
+```
+
+The file `src/language-server/hello-world.langium` in your newly created project contains your grammar.
+
+## Explaining the terms
+
+If you're already familiar with the terms used in parsing or DSL frameworks, you can skip this short excursion and go straight to the next part. However, anyone who is new to DSL development should carefully read the following primer on the terms we are using in our documentation:
+
+_abstract syntax tree_: A tree of elements that represents a text document in an object-oriented form. Commonly abbreviated as _AST_.
```

Comment:
Why _object-oriented_? I'd rather highlight that this data structure abstracts from the actual text representation by combining multiple tokens into one JS object.